packages/amplify/amplify_flutter/pubspec.yaml

```diff
@@ -43,7 +43,7 @@ dev_dependencies:
   build_runner: ^2.0.0
   flutter_test:
     sdk: flutter
-  json_serializable: ^6.3.0
+  json_serializable: 6.3.1
```

Comment:
Yes, these should be locked since changes will cause the generated outputs to change which will impact tests which check for changes. Best for them all to be bumped at once across all packages.